recipes/tensorflow-lite/all/CMakeLists.txt

```diff
@@ -1,8 +1,44 @@
 cmake_minimum_required(VERSION 3.1)
 project(cmake_wrapper)
 
-include(conanbuildinfo.cmake)
+include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
 conan_basic_setup(KEEP_RPATHS)
 
-add_subdirectory("src/tensorflow/lite")
+find_package(Eigen3 REQUIRED)
+find_package(absl REQUIRED)
+
+add_library(NEON_2_SSE INTERFACE IMPORTED)
+if (NOT CMAKE_SYSTEM_PROCESSOR OR CMAKE_SYSTEM_PROCESSOR MATCHES "x86")
+    find_package(NEON_2_SSE REQUIRED)
+    set_property(TARGET NEON_2_SSE PROPERTY INTERFACE_LINK_LIBRARIES NEON_2_SSE::NEON_2_SSE)
+endif()
+
+find_package(farmhash REQUIRED)
+add_library(farmhash INTERFACE IMPORTED)
+set_property(TARGET farmhash PROPERTY INTERFACE_LINK_LIBRARIES farmhash::farmhash)
```

Comment:
Why not use alias? https://cmake.org/cmake/help/latest/command/add_library.html#alias-libraries